### ISSUE TITLE
ipv6 not disabled until reboot

### ISSUE THREAD

**Issue by lo5an** [OULibraries/ansible-role-centos7#21]
It looks like ipv6 is configured to be disabled on reboot, but not turned off in the running system.

**Comment by jsnshrmn** [comment]
@lo5an on a fresh box, can you run
```
sudo sysctl -n net.ipv6.conf.all.disable_ipv6
```
and tell me the output?  If the value is zero, then I'll add
```
sysctl -w net.ipv6.conf.all.disable_ipv6=1
```
to the virtualbox playbook in the role to force an immediate change

**Comment by jsnshrmn** [comment]
I just tested this and verified that the value is 0 on first boot.  Implementing fix now.

**Comment by jsnshrmn** [comment]
verified closed by commit  98d7a62
